### DISCUSSION TITLE
Режим стримера/режим полного погружения.

### DISCUSSION THREAD

**Discussion by SemjonBelus** [octothorp-team/public-dobrograd#727] [Предложения]
### Всех приветствую!
![image](https://user-images.githubusercontent.com/93833754/141342611-b008b621-a55c-470d-974e-3115c61c5ebd.png)
Я люблю снимать видео с ролевой игрой... НО КАК ЖЕ блин раздражают механические сообщения, которых в миллион раз больше количества ролевых команд. Это очень сильно сбивает с ролевой игры и я предлагаю исправить это!

**1) В "режиме стримера" убрать подобные сообщения:**
Семён Белоусов на пути в город Доброград; 
Зарплата, ты получил 500Р; 
Ты проголодался, пора перекусить;
Семён Белоусов арестован на 1200 секунд по причине: не заплатил за стажировку;
[#] Жалоба закрыта;
В ближайшее время с твоего банковского счёта будет списано 120Р за аренду...;
[#] Прицел включён/выключен;
Банк пишет тебе СМС;
Быть примерным гражданином просто, правда? (текущая карма: 383)
И так далее (могу позже привести полный список)

**2) Увеличить всем игрокам КД на рекламу до 5-10 минут, увеличить её стоимость.**

**3) В "режиме стримера" сократить некоторые важные элементы игры, которые способны повлиять на ролевую игру:**
Итан Вилсон подал в розыск Семёна Белоусова. Причина: зафрикиллил при свидетелях —> Семён Белоусов в розыске. Зафрикиллил при свидетелях.
Семён Белоусов больше не разыскивается. Розыск отозвал: Итан Вилсон. Причина: арестован —> Семён Белоусов не разыскивается. Арестован.

**4) Убрать урчание живота при низком голоде**

**Comment by KOwOMei** [comment]
Вообще можно убрать сообщения про вход людей с помощью F4 - настройки. Это так, к слову.
